### DISCUSSION TITLE
Welcome to mdk2 Discussions!

### DISCUSSION THREAD

**Discussion by malware-dev** [malforge/mdk2#18] [Announcements]
## Hey there!

Feel free to ask any questions – **no honest question is ever stupid**. Share your ideas, interact with other members, and be welcoming and open-minded. This is a community we're building together. To get started, comment below with a bit about yourself and what you do in this community.

Remember to be respectful and considerate in your interactions. We're all here to learn and grow together, so let's keep the discussions positive and constructive.

I will often be reachable on [Keen's Discord](https://discord.com/invite/keenswh), depending on the time zone you're in.

**Comment by Aimon94** [comment]
I would like to chat with you on a messenger and ask you questions about programming, and maybe even become friends. If you don't like it, don't take it personally, just ignore it.  (discord/aimon94)

**Reply by malware-dev** [reply]
I'd recommend asking questions in the #programmable-block channel on the Keen discord. You can often find me there too, and everybody is rather welcoming. We often answer questions just on plain programming as well as SE specific scripting.

I don't add people to my friends list that I don't already know in some measure, it's nothing personal.

**Comment by BelaSchrammDev** [comment]
## Hi guys. 👋

I'm Béla, 55 years old, living in Leipzig (Germany) and a very big Space Engineers fan. I'm especially fascinated by the idea of working with ingame scripts. 
That's why I think the MDK project is super fantastic. Thanks for that. 👍
Can I send you something as a thank you for your efforts?

Programming has been with me my whole life. I've just completed further training as a web developer and want to work in this field. But it's a bit difficult to find a job as a career changer at the moment. But there's no giving up. 💪

I also have two scripts in WorkShop:

https://steamcommunity.com/sharedfiles/filedetails/?id=2080547657

https://steamcommunity.com/sharedfiles/filedetails/?id=1481314096

I will rewrite the first one with MDK2. It wouldn't be so nice without your MDK. Thanks again. 

Best regards 
Béla.

**Reply by malware-dev** [reply]
Hello and welcome, Béla! I am very glad you find this useful.


> Can I send you something as a thank you for your efforts?

You just did 😉 A little bit of gratitude goes a long way 😁

**Comment by dotSp0T** [comment]
Just got back into playing SE and wanted to load my old scripts. Saw that you released a new version of this awesome toolkit. Thank you so much for your work!

**Reply by malware-dev** [reply]
You're welcome. Hope the new kit will serve you well.
